packages/aws-fargate/src/preactivate.js

```diff
@@ -11,7 +11,8 @@ if (isNodeJsTooOld()) {
   // eslint-disable-next-line no-console
   console.error(
     `The package @instana/aws-fargate requires at least Node.js ${minimumNodeJsVersion} but this process is ` +
-      `running on Node.js ${process.version}. This Fargate container will not be monitored by Instana.`
+      `running on Node.js ${process.version}. This Fargate container will not be monitored by Instana.` +
+      'See https://www.ibm.com/docs/en/instana-observability/current?topic=agents-aws-fargate#versioning.'
```

Comment:
It is a good idea to link there 👍  Maybe we should also add a table to that section of the docs that explains which image version can be used for which Node.js version?

Reply:
Yes we will update the docs as part of the v3 release 👍